src/screens/SSICredentialsRequiredScreen/index.tsx

```diff
@@ -50,10 +53,17 @@ const SSICredentialsRequiredScreen: FC<Props> = (props: Props): JSX.Element => {
           restrictToFormats: format,
           restrictToDIDMethods: subjectSyntaxTypesSupported,
         });
-        const matchedVCs: Array<UniqueVerifiableCredential> = selectResult.verifiableCredential
-          ? selectResult.verifiableCredential
-              .map((matchedVC: IVerifiableCredential) => getMatchingUniqueVerifiableCredential(uniqueVCs, matchedVC))
-              .filter((matchedVC): matchedVC is UniqueVerifiableCredential => !!matchedVC) // filter out the undefined (should not happen)
+        if (selectResult.areRequiredCredentialsPresent === "error") {
+          console.info('pex.selectFrom returned errors:\n', JSON.stringify(selectResult.errors));
+        }
+        const matchedVCs: Array<UniqueVerifiableCredential> = selectResult.matches && selectResult.verifiableCredential
+          ? selectResult.matches.map((match: SubmissionRequirementMatch) => {
+            const matchedVC = JSONPath.query(selectResult, match.vc_path[0])
+            if(matchedVC && matchedVC.length) {
```

Comment:
can be shortened by

if (matchedVC?.lenght) {

Reply:
done

Reply:
@nklomp technically matchedVC?.lenght is only true for 1 orr more. but i do agree that using > 0 is more readable

Reply:
Nit going to approve this change

Reply:
We went from  something not open to any misinterpretation to something that is open to it. The fact you can use numbers in truthy calls to your advantage doesn't mean you should.  So don't please

Reply:
@nklomp 
I changed it to a while ago
```
if (matchedVC?.length > 0) {
```
Isn't that ok?

Reply:
Just leave it as it was. That expression was done like that very much on purpose. You are now still relying on type coallescence in your bigger than expression. Basically resulting in undefined > 0. Which is false and thus correct, but still less readable then the original. Clarifty is always important and I simply don't like that we would go from something very clear to something less clear. Makes no sense at all to me